### PR TITLE
feat(fix bug , add nexttime feature):

### PR DESCRIPTION
**Pull request by kingeasternsun**
1. fix bug:    "Mon/2",  job should run every two days from Monday, that is  Monday,Wednesday,Friday
2. add function to get the next time to run

1.修复bug： “Mon/2” ，意味着job从周一每隔两天执行一次，也就是星期一，星期三，星期五
2. 添加函数获取任务下次执行时间

### CODE REVIEW

**Comment by gqcn** [comment]
@kingeasternsun 请增加针对修复问题的单侧，增加代码注释（可以从`cron`项目中复制）。

**Comment by gqcn** [comment]
@kingeasternsun CI失败了，麻烦瞅瞅呢

File: os/gcron/gcron_schedule.go [hunk 1]
```diff
@@ -263,3 +264,115 @@ func (s *cronSchedule) meet(t time.Time) bool {
 		return true
 	}
 }
+
+// 参考 robfig/cron的方法计算 github.com/robfi/cron/v3@v3.0.0/spec.go
```
Comment (gqcn) [inline os/gcron/gcron_schedule.go:268]: Please use English comment in codes.

File: os/gcron/gcron_schedule.go [hunk 2]
```diff
@@ -189,7 +188,9 @@ func parsePatternItem(item string, min int, max int, allowQuestionMark bool) (ma
 					return nil, gerror.NewCodef(gcode.CodeInvalidParameter, `invalid pattern item: "%s"`, item)
 				} else {
 					rangeMin = number
-					rangeMax = number
```
Comment (gqcn) [inline os/gcron/gcron_schedule.go:192]: You'd better commit an unit testing case for this update.

**Review by gqcn — CHANGES_REQUESTED** [review]

File: os/gcron/gcron_schedule.go (hunk 1 above)
Comment (kingeasternsun) [reply os/gcron/gcron_schedule.go]: ok, English comment has added

File: os/gcron/gcron_schedule.go (hunk 2 above)
Comment (kingeasternsun) [reply os/gcron/gcron_schedule.go:192]: The TestFunction of  "TestSlash"  in file gcron_schedule_test.go  is the unit testing for this update

**Pull request merged**